packages/core/src/Stores/contract-trade-store.js

```diff
@@ -397,7 +397,7 @@ export default class ContractTradeStore extends BaseStore {
 
     get last_contract() {
         const applicable_contracts = this.applicable_contracts();
-        const length = applicable_contracts.length;
+        const length = !!this.contracts.length && applicable_contracts.length;
```

Comment:
@kate-deriv I'm not sure what are you doing here. If `this.contracts.length === 0` then in result `length` will be equal to `false`, it's a `boolean` value not `number`.

Reply:
@sergei-deriv it's equally ok if it's false or 0 because we just need to check the length, and if it's 0 or false, we need to return {} as last_contract value. It's up to you @kate-deriv :) It will work with or without `!!`